library/src/main/java/com/getstream/sdk/chat/rest/core/Client.java

```diff
@@ -1109,11 +1109,15 @@ public void updateMessage(@NonNull Message message,
 
             @Override
             public void onResponse(Call<MessageResponse> call, Response<MessageResponse> response) {
+                message.setSyncStatus(SYNCED);
+                if (response.body() != null && response.body().getMessage() != null)
```

Comment:
We need to log if there is no `body` and `message`. That's an exceptional case. And call `callback.onError`.
Also could be wrapped into some `ResponseUtilities` like as it is repeated:
```java
ResponseUtilities.setMessageStatus(response, SYNCED);
```